src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/CreateCohortController.cs

```diff
@@ -88,21 +98,57 @@ public async Task<IActionResult> SelectProvider(SelectProviderViewModel request)
                     ModelState.AddModelError(nameof(request.ProviderId), "Check UK Provider Reference Number");
                     return View(request);
                 }
-                _logger.LogError($"Failed '{nameof(CreateCohortController)}-{nameof(SelectProvider)}': {nameof(ex.StatusCode)}='{ex.StatusCode}', {nameof(ex.ReasonPhrase)}='{ex.ReasonPhrase}'");
+
+                _logger.LogError(
+                    $"Failed '{nameof(CreateCohortController)}-{nameof(SelectProvider)}': {nameof(ex.StatusCode)}='{ex.StatusCode}', {nameof(ex.ReasonPhrase)}='{ex.ReasonPhrase}'");
             }
             catch (Exception ex)
             {
-                _logger.LogError($"Failed '{nameof(CreateCohortController)}-{nameof(SelectProvider)}': {nameof(ex.Message)}='{ex.Message}', {nameof(ex.StackTrace)}='{ex.StackTrace}'");
+                _logger.LogError(
+                    $"Failed '{nameof(CreateCohortController)}-{nameof(SelectProvider)}': {nameof(ex.Message)}='{ex.Message}', {nameof(ex.StackTrace)}='{ex.StackTrace}'");
```

Comment:
Should we re-throw an un-handled exception?

Reply:
Not sure this needs re-throwing - its handled and logged, it will fall through to the `RedirectToAction("Error","Error")` at the bottom of the action